cmd/prometheus/main.go

```diff
@@ -204,6 +204,7 @@ func Main() int {
 	prometheus.MustRegister(notifier)
 	prometheus.MustRegister(configSuccess)
 	prometheus.MustRegister(configSuccessTime)
+	prometheus.MustRegister(queryEngine)
```

Comment:
Should I keep the "maxConcurrentQueries"? And, in that case, register it as a gauge (instead of using MustConstMetric) as I will need to set its value when creating the Engine?